### ISSUE TITLE
Sorting by name - ToDo Explorer

### ISSUE THREAD

**Issue by SonGokussj4** [rubberduck-vba/Rubberduck#4904]
**Justification**
Allow Todo Explorer to sort results (in By Marker) by Description name.

**Description**
Best would be the classic behaving - click on a column name and it would sort it ascending/descending. But I know/think that would be a lot of work.

So for now, would it be possible to add sorting by Description? I don't know what sort is applied here if any. Not Line Numbers, not Module Name and not Description.

I'm asking because I'm trying to add 
TODATE 2018-05-21 update this
TODATE 2018-05-12 update that
TODATE 2018-05-28 update everything

And now it's not respecting any order.

**Additional context**
Ordering by description would sort my "TODATE" dates :-)
![image](https://user-images.githubusercontent.com/13331858/55692462-96bf2d80-59a8-11e9-91f7-c10f3cb7cdaf.png)

**Comment by retailcoder** [comment]
Not sure to what extent the GroupingGrid interferes with that, but custom sorting with clickable headers isn't really hard; the ViewModel properties only need to implement IComparable - and being strings, we get it for free: likely all that's missing is `CanUserSortColumns="true"` in the markup for the grid.
I believe the GroupingGrid would keep the groupings and sort by the selected column within each group.

**Comment by retailcoder** [comment]
Side note, comments should say *why*, not *what*: the history of what happened when should be recorded in source control, not in code. Consider regularly exporting the code files to a local git repository and dating changesets with actual commits.

**Comment by SonGokussj4** [comment]
I agree that I should not have "TODO 2019-04-08 Need to explain my todo comments" in my code.

This is a special case for this project because my superior wants to know when did I think about changing what or when did I create the "todo". This wasn't possible without RD. But now it is so I'm using it :-)

If the implementation is easy, I would be happy if you consider it.

**Comment by bclothier** [comment]
@IvenBach  was this addressed in your PR #4986 ? If so, this should be closed as fixed?

**Comment by IvenBach** [comment]
No. My PR only deals with reordering the columns headers. The order of the results and grouping has not been changed.

**Comment by SonGokussj4** [comment]
Sad face :-)
If I knew how, I would try to do it myself. But it's higher level for me :-/
